numba/np/random/distributions.py

```diff
@@ -574,3 +574,164 @@ def random_logseries(bitgen, p):
             return 1
         else:
             return 2
+
+
+@register_jitable
+def random_binomial_btpe(bitgen, n, p):
+    r = min(p, 1.0 - p)
+    q = 1.0 - r
+    fm = n * r + r
+    m = np.floor(fm)
```

Comment:
Think this should be cast to the Numba equivalent of `RAND_INT_TYPE`?

Reply:
Resolved in [3dfb3ca](https://github.com/numba/numba/pull/8371/commits/3dfb3cad5a2629fe52672ad71534e215ee3063e1)